### PR TITLE
Add font ligatures support.

### PR DESCRIPTION
**Pull request by pmros**
[Review on Reviewable](https://reviewable.io/reviews/princejwesley/mancy/52)

### CODE REVIEW

**Comment by pmros** [comment]
=>
mancyligatures

**Comment by princejwesley** [comment]
@pmros It looks good. I'll try this out before merging this PR.  We have to give provision to select fonts from preference panel.

File: fonts/FiraCode_Regular/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+## Fira Code: monospaced font with programming ligatures
```
Comment (princejwesley) [inline fonts/FiraCode_Regular/README.md]: @pmros do we need readme file?

**Comment by princejwesley** [comment]
@pmros Remove readme.md and give PR against `fonts-integration` branch. we can experiment/add fonts and system font access from preferences.

**Comment by pmros** [comment]
Done!


---







Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/princejwesley/mancy/52)

**Comment by pmros** [comment]
Another pull request added to fonts-integration branch

**Pull request closed without merging**